x-pack/plugins/ml/public/file_datavisualizer/components/edit_flyout/overrides_validation.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+const INDETERMINATE_FIELD_PLACEHOLDER = '?';
```

Comment:
Is this character ever likely to change from being a question mark character? If not, I don't think this needs to be defined as a `const` - see comment about translation below.

Reply:
No, it doesn't change. I think it's fine to just put it in as a regular question mark 👍 
Done in 51c237651d7a6baebf826a47672823461f8055d9